circle.yml

```diff
@@ -3,20 +3,20 @@ machine:
     LANG: en_US.UTF-8
     LC_CTYPE: en_US.UTF-8
   xcode:
-    version: "6.2"
+    version: "6.3.1"
 dependencies:
   pre:
-    - xcrun instruments -w "iPhone 6 (8.2 Simulator)" || exit 0
+    - xcrun instruments -w "iPhone 6 (8.3 Simulator)" || exit 0
   override:    
-    - sudo gem install cocoapods xcpretty obcd slather -N
-    - pod install --project-directory=Example
+    - bundle check --path=vendor/bundle || bundle install --path=vendor/bundle --jobs=4
```

Comment:
I don't think CircleCI's caching is enabled yet for OS X builds, but if it were, `bundle check` would make for a faster build by skipping bundle install in the event that the dependencies are cached from a previous build (and none of them changed)